package.json

```diff
@@ -27,16 +27,20 @@
     "test": "npx hardhat test",
     "coverage": "npx hardhat coverage",
     "format": "npx prettier --write",
-    "check-format": "prettier --check '{scripts,test,contracts}/**/*.{js,sol}'",
+    "check-format": "prettier --check '{scripts,test,contracts}/**/*.{ts,sol}'",
```

Comment:
I think we should have the same config (`'{scripts,test,...'`) for both tasks `format` and `check-format`. We can either use it for both or remove it for both.